PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -556,14 +561,18 @@ export class ParaReportPublishService {
     private async processSarifResults(): Promise<void> {
         if (this.sarifReports.length > 0) {
```

Comment:
@yshi-parasoft The current approach only updates SARIF reports in artifacts when there are converted SARIF reports in the current task and changes to the reference build. Should we consider updating SARIF reports in artifacts when **no static report is specified** and  changes to the reference build in the current task?